lib/experimental/new_embed.dart

```diff
@@ -1093,43 +1131,88 @@ class ConsoleExpandController extends ConsoleController {
 enum DialogResult {
   yes,
   no,
+  ok,
+  cancel,
```

Comment:
We should consider consolidating these. yes/no and ok/cancel have the same effect.

Reply:
In other places, I've generally seen the values in enums like this broken out (mostly because you end up with a bunch more of them anyway).

Windows does it that way, for instance:

https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.dialogresult?view=netframework-4.8

Still, if we don't have the need for any additional result types in the next few months, it could be a smart refactor.